Synfig-core fails to build libavcodec module with old FFmpeg, no video import feature

System: Ubuntu 16.04

When trying to build synfig-core using "--with-libavcodec" option the compilation fails with an error:

```
/synfig/synfig-core/src/modules/mod_libavcodec/trgt_av.cpp:199:6: error: reference to ‘AVPixelFormat’ is ambiguous
synfig/synfig-core/src/modules/mod_libavcodec/trgt_av.cpp:209:53: error: ‘AVStream {aka struct AVStream}’ has no member named ‘codecpar’
```

Currently libavcodec module is disabled (see #742 and #743), but with this we also lost support for video import. This should be solved at high priority.


<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/70886066-synfig-core-fails-to-build-libavcodec-module-with-old-ffmpeg-no-video-import-feature?utm_campaign=plugin&utm_content=tracker%2F321991&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F321991&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/synfig-core/configure.ac
+++ b/synfig-core/configure.ac
@@ -166,7 +166,7 @@ AC_ARG_WITH(libavcodec,
 ])
 
 if test $with_libavcodec != "no" ; then {
-	PKG_CHECK_MODULES(LIBAVCODEC, [libavcodec >= 51.42 libavformat >= 54.0 libavutil >= 52.9 libswscale >= 2.1.2],
+	PKG_CHECK_MODULES(LIBAVCODEC, [libavcodec >= 57.0 libavformat >= 57.0 libavutil >= 55.0 libswscale >= 4.0],
 		[CONFIG_DEPS="$CONFIG_DEPS libavcodec libavformat libswscale"],
 		[echo no; with_libavcodec="no"])
 } ; fi

